app/src/main/java/ca/fuwafuwa/kaku/Windows/CaptureWindow.java

```diff
@@ -56,7 +57,13 @@ public boolean onTouch(MotionEvent e) {
     @Override
     public boolean onDoubleTap(MotionEvent e) {
         tessStarted = true;
-        mTessRunnable.runTess(new BoxParams(params.x, params.y + getStatusBarHeight(), params.width, params.height));
+
+        int[] viewPos = new int[2];
+        mWindowBox.getLocationOnScreen(viewPos);
+
+        // TODO: Replace the first 1 with R.drawable.border_translucent.StrokeWidth when I figure out how
+        int offset = KakuTools.dpToPx(this.context, 1)+1;
+        mTessRunnable.runTess(new BoxParams(viewPos[0]+offset, viewPos[1]+offset, params.width-(2*offset), params.height-(2*offset)));
```

Comment:
> How in the world do you click that fast? I have been trying to duplicate this for like 10 minutes and have not even once seen it.

Ops, sorry I forgot to run the app again in Android Studio. You're right, looks like double tap is fairly consistent on the main body of the capture window. We might still hit a issue if we double tap on the resize view:

![screen 1708226508026677](https://cloud.githubusercontent.com/assets/878158/22077735/468e64e6-dd6a-11e6-9e8a-b185db08855f.png)

![screen 1708242869447608](https://cloud.githubusercontent.com/assets/878158/22077812/9881b604-dd6a-11e6-98c7-bfdf0d958272.png)